website/docs/r/cis_healthcheck.html.markdown

```diff
@@ -30,44 +30,45 @@ resource "ibm_cis_healthcheck" "test" {
+- `follow_redirects`-(Optional,bool) Follow redirects if returned by the origin.
+- `allow_insecure`-(Optional,bool) Do not validate the certificate when healthcheck use HTTPS.
+- `description` - (Optional,string) Free text description.The Default value is false.
+- `port` - (Optional,int) The TCP port to use for the health check.
 
 [`expected_body`],[`expected_codes`] are required aruguments when the type is HTTP or HTTPS.
 
-Header is not currently supported in this version of the provider. 
+Header is not currently supported in this version of the provider.
 
 ## Attributes Reference
 
 The following attributes are exported:
 
-* `id` - Load balancer monitor ID.
-* `created_on` - The RFC3339 timestamp of when the load balancer monitor was created.
-* `modified_on` - The RFC3339 timestamp of when the load balancer monitor was last modified.
+- `id` - Load balancer monitor ID.
+- `monitor_id` - Load balancer monitor ID.
+- `created_on` - The RFC3339 timestamp of when the load balancer monitor was created.
+- `modified_on` - The RFC3339 timestamp of when the load balancer monitor was last modified.
 
 ## Import
 
-The `ibm_cis_health_check` resource can be imported using the `id`. The ID is formed from the `Healthcheck Id` and the `CRN` (Cloud Resource Name) concatentated usinga `:` character.  
+The `ibm_cis_health_check` resource can be imported using the `id`. The ID is formed from the `Healthcheck Id` and the `CRN` (Cloud Resource Name) concatentated usinga `:` character.
 
-The CRN will be located on the **Overview** page of the Internet Services instance under the **Domain** heading. 
+The CRN will be located on the **Overview** page of the Internet Services instance under the **Domain** heading.
 
-* **CRN** is a 120 digit character string of the form: `crn:v1:bluemix:public:internet-svcs:global:a/4ea1882a2d3401ed1e459979941966ea:31fa970d-51d0-4b05-893e-251cba75a7b3::`
-
-* **Healthcheck ID** is a 32 digit character string of the form: `1fc7c3247067ee00856729661c7d58c9`. The id of an existing Healthcheck monitor is not avaiable via the UI. It can be retrieved programmatically via the CIS API or via the CLI using the CIS command to list the defined GLBs:  `bx cis glb-monitors` 
+- **CRN** is a 120 digit character string of the form: `crn:v1:bluemix:public:internet-svcs:global:a/4ea1882a2d3401ed1e459979941966ea:31fa970d-51d0-4b05-893e-251cba75a7b3::`
 
+- **Healthcheck ID** is a 32 digit character string of the form: `1fc7c3247067ee00856729661c7d58c9`. The id of an existing Healthcheck monitor is not avaiable via the UI. It can be retrieved programmatically via the CIS API or via the CLI using the CIS command to list the defined GLBs: `ibmcloud cis glb-monitors`
 
 ```
-$ terraform import ibm_cis_healthcheck.myorg <healthcheck_id>:<crn>
+$ terraform import ibm_cis_healthcheck.myorg <monitor_id>:<crn>
```

Comment:
monitor_id will be confusing. instead name it id.
User might think that its not the combination, as monitor_id doesn't refer to combination in our docs

Reply:
done